Added first revised version of performance graph.

tobiasraabe opened the pull request:
Now, I included the png to simplify the lookup. You proposed a vertical line for the KW1994 solution but I found it to be confusingly similar to the bars which is not correct, since they do not share the same y axis. Therefore, I created a black point at the result and connected it to the correct axes. Is this more comprehensible and pretty?

peisenha commented:
Thanks a lot, this is done from your side. I will integrate it in the manuscript in the next couple of days ....

tobiasraabe commented:
Hey Philipp,

I added the second graph in another notebook.

I also tried to adjust the code for the model solution, but I could not load the pickle file. It states ``ImportError: No module named datetime`` which is odd because I had no problems before. Maybe it is related to https://stackoverflow.com/a/22428308/7523785. I updated my anaconda 2.7 distribution between the graphs and that might have changed paths.

peisenha commented:
Great, thanks a lot!
